.gitignore

```diff
@@ -33,6 +33,7 @@ MANIFEST
 # Static files
 polaris/polaris/collectstatic/*
 !polaris/polaris/collectstatic/__init__.py
+polaris/polaris/static/base.css
```

Comment:
This file is now created via `python manage.py compilescss` so we don't want to check it into version control